### DISCUSSION TITLE
Malware and Spyware after a factory reset

### DISCUSSION THREAD

**Discussion by ghost** [community/community#132095] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

hello everyone! I have some serious issues with my cell phone being full of malware and RAT's and spyware, even after a factory reset, and I am unsure if my chromebook has the same issue, I recently did a powerwash on the chromebook. I'm seeking help to fix these issues, I honestly do not even know where to begin. I would be grateful for any help with this. Thank you! ~ John

**Comment by nadeeshaofficial1** [comment]
1. Factory Reset
   - Go to your phone's Settings.
   - Look for 'System' or 'General Management.'
   - Find 'Reset' or 'Backup & Reset.'
   - Choose 'Factory data reset' and confirm.
   - Do not restore apps immediately after the reset.

2. Install Antivirus
   - Download a trusted antivirus app from the Google Play Store.
   - Install and run a full scan of your phone.

3. Review App Permissions
   - Go to Settings > Apps & notifications > App permissions.
   - Review and revoke permissions for suspicious or unnecessary apps.

4. App Installation
   - Only download apps from the Google Play Store.
   - Avoid sideloading apps from unknown sources.

5. Update Software
   - Go to Settings > System > Software Update.
   - Check for and install any available updates for your phone.

For Your Chromebook

1. Perform Powerwash
   - Open Settings from the Chrome menu (three dots).
   - Scroll down and click 'Advanced.'
   - Under 'Reset settings,' click 'Powerwash' and confirm.

2. Use the Chrome Cleanup Tool
   - Open Chrome browser.
   - Click on the three dots menu > Settings.
   - Scroll down and click on 'Advanced.'
   - Under 'Reset and clean up,' click 'Clean up computer.'
   - Follow the prompts to remove unwanted extensions and reset browser settings.

3. Check for Chrome OS Updates
   - Click on the bottom right corner where your account picture is located.
   - Click 'Settings.'
   - Scroll down and click 'About Chrome OS.'
   - Click 'Check for updates' and install any available updates.

4. Install Apps from Trusted Sources
   - Use the Chrome Web Store for installing extensions and apps.
   - Please avoid third-party sources unless you need to and trusted them.

5. Password and Account Security
   - Change passwords for critical accounts using a secure device.
   - Enable Two-Factor Authentication (2FA) for added security.

**Reply by ghost** [reply]
Thanks so much for your help! Part of the issue is I have tried these steps and when I go into my apps setting, i click the top right icon and it "Shows system apps or permissions" and then literelly hundreds of apps show up that have permissions to basically do anything with my cell phone. It can change and monitor my wifi, prevent phone from sleeping when i close the phone, read and make changes to contacts, basically anything you tjhat would be harmful or intrusive to my phone, it can do. And I cannot disable these apps, it wont allow me to, the option is greyed out. I am so frustrated with this, its a brand new phone, It's been like this since the day I bought it. I literelly don't know whsat to do.

**Comment by catunderrain** [comment]
Hi John, I see your problem, you can try these steps: 
- Backup all essential data to a secure cloud service or an external device.
- Factory Reset again just to ensure it's done correctly.
- Now go to Secure Boot Mode, for most phones: Press and hold the power button, then tap and hold "Power off" until you see the option to reboot to safe mode.
- In safe mode, go to Settings > Apps, look for any unfamiliar, suspicious apps and uninstall them. 
- (additionally) Download a reputable security app like Avast, Malwarebytes, or Bitdefender from the official app store and perform a full scan. 

Hope you get well with your issue, my friend!

**Reply by ghost** [reply]
Thanks so much for your help! Part of the issue is I have tried these steps and when I go into my apps setting, i click the top right icon and it "Shows system apps or permissions" and then literelly hundreds of apps show up that have permissions to basically do anything with my cell phone. It can change and monitor my wifi, prevent phone from sleeping when i close the phone, read and make changes to contacts, basically anything you tjhat would be harmful or intrusive to my phone, it can do. And I cannot disable these apps, it wont allow me to, the option is greyed out. I am so frustrated with this, its a brand new phone, It's been like this since the day I bought it. I literelly don't know whsat to do.

**Reply by catunderrain** [reply]
I thought you should bring it to the authorized store for repairs, or contact the brand for a replacement. GL bro

**Comment by danialtavakoli** [comment]
Hello John,

I'm sorry to hear about the issues you're experiencing with your devices. Malware, RATs (Remote Access Trojans), and spyware can be persistent and difficult to remove, but there are steps you can take to secure your devices. Here’s a comprehensive guide to help you clean your phone and Chromebook:

### Securing Your Cell Phone

1. **Back Up Important Data**:
   - Before proceeding, ensure you back up important data such as contacts, photos, and documents to a secure location, such as Google Drive or an external hard drive.

2. **Factory Reset**:
   - Perform another factory reset, but this time do not restore from a backup immediately. Start fresh to ensure no malware is reintroduced.

3. **Install a Reliable Mobile Security App**:
   - Download a trusted security app from the Google Play Store or Apple App Store, such as Malwarebytes, Avast, or Bitdefender.
   - Run a full scan to identify and remove any malware.

4. **Update OS and Apps**:
   - Ensure your phone's operating system and all apps are up to date, as updates often include security patches.

5. **Be Cautious with App Installations**:
   - Only install apps from official app stores (Google Play Store or Apple App Store).
   - Avoid sideloading apps or installing from unknown sources.
   - Review app permissions and ensure they make sense for the app's functionality.

6. **Enable Security Features**:
   - Enable Google Play Protect on Android.
   - Use built-in security features like face recognition, fingerprint scan, or a strong password/PIN.

### Securing Your Chromebook

1. **Back Up Data**:
   - Make sure to back up any important data to Google Drive or an external storage device.

2. **Powerwash (Factory Reset)**:
   - Perform a powerwash on your Chromebook again to reset it to factory settings. This can be done by going to Settings > Advanced > Reset Settings > Powerwash.

3. **Update Chrome OS**:
   - Ensure your Chromebook is running the latest version of Chrome OS by going to Settings > About Chrome OS > Check for updates.

4. **Install Antivirus Extensions**:
   - Consider adding trusted antivirus or security extensions from the Chrome Web Store, such as Malwarebytes Browser Guard or Avast Online Security.

5. **Enable Safe Browsing**:
   - Go to Chrome Settings > Privacy and security > Security and enable Safe Browsing (Enhanced Protection).

6. **Review Extensions and Apps**:
   - Regularly review installed extensions and apps. Remove any that you do not recognize or no longer need.

7. **Enable Verified Boot**:
   - Chrome OS has a feature called Verified Boot that checks the integrity of the operating system. Ensure it is enabled by default, which helps protect against tampering.

### Additional Security Measures

1. **Change Passwords**:
   - Change passwords for all important accounts, especially those accessed from the infected devices. Use strong, unique passwords for each account and consider using a password manager.

2. **Enable Two-Factor Authentication (2FA)**:
   - Enable 2FA on all accounts that support it for an added layer of security.

3. **Monitor for Suspicious Activity**:
   - Keep an eye on your accounts and devices for any unusual activity. Report and secure any compromised accounts immediately.

4. **Regular Scans and Updates**:
   - Regularly scan your devices with security software and keep all software, including apps and OS, up to date.

By following these steps, you should be able to secure your cell phone and Chromebook from malware, RATs, and spyware. If issues persist, consider seeking professional help from a trusted tech support service.
